telegram/ext/_application.py

```diff
@@ -133,11 +133,13 @@ class Application(Generic[BT, CCT, UD, CD, BD, JQ], AbstractAsyncContextManager)
         finally:
             await application.shutdown()
 
```

Comment:
I did not include the code above into the "Examples" block because it looks more like an explanation than an example.